.github/workflows/qemu-jobs.yml

```diff
@@ -0,0 +1,193 @@
+name: Build and check tests for QFlex enabled and base QEMU
+
+on: 
+  push:
+    branches:
+      - master
+      - dev
+  pull_request:
+    branches:
+      - master
+      - dev
+  workflow_dispatch:
+
+jobs:
+  build_and_check_job_definition: # native_build_job_definition:
+    runs-on: ubuntu-latest
+    name: Configure, build and check QEMU # native_build_job_definition
+    strategy:
+      matrix:
+        include:
+          - # NOTE: --enable-fdt=system --enable-slirp=system was removed
+            build-name: Build for Ubuntu 20.04 + check # build-system-ubuntu
+            targets: aarch64-softmmu
+            make-check-args: check
+          - build-name: Build for user-mode + check-tcg # build-user
+            targets: aarch64-linux-user
+            configure-args: --disable-tools --disable-system
+            make-check-args: check-tcg
+          - build-name: Build for system-mode + check-tcg # build-some-softmmu
```

Comment:
Looking at the test output, `check-tcg` runs nothing. All the tests are skipped.

Are you sure that `check-tcg` is compatible with full system emulation?

Reply:
Resolved in https://github.com/parsa-epfl/qemu/pull/62#issuecomment-882524912